### PR TITLE
Add custom header to Nightcrawl

### PR DESCRIPTION
**Pull request by weitzman**
**Jira:** (Skip unless you are MA staff)
DP-28076

**To Test:**
- [ ] I ran `ahoy nightcrawl` and stepped through Drupal in the debugger until I could see $request in index.php. Then look in the debug variables for $request['headers']. This passes once `MASS_BYPASS_RATE_LIMIT` is present.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8d91c36f2c9b6ffd28c1f740b31253c74083b233
</p>

File: changelogs/DP-28706.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Added:
+  - description: Add custom header to Nightcrawl
```
Comment (tom-fleming) [inline changelogs/DP-28706.yml]:
```suggestion
  - description: Changed format of bypass header and added to Nightcrawler
```

File: changelogs/DP-28706.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# You may add more than 1 description & issue for each type using the following format:
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Added:
```
Comment (tom-fleming) [inline changelogs/DP-28706.yml]:
```suggestion
Changed:
```

**Comment by weitzman** [comment]
Thanks for tracking down this nginx nonsense. Presumably this requires an Akamai change as well.

File: .circleci/nightcrawler/nightcrawler.js [hunk 3]
```diff
@@ -69,6 +69,9 @@ function getAuth() {
 
 var driver = new RequestDriver({
   strictSSL: false,
+  headers: {
+    "mass-bypass-rate-limit": process.env.MASS_BYPASS_RATE_LIMIT,
```
Comment (tom-fleming) [inline .circleci/nightcrawler/nightcrawler.js:73]:
```suggestion
    "Mass-Bypass-Rate-Limit": process.env.MASS_BYPASS_RATE_LIMIT,
```

File: backstop/scripts/before.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ module.exports = async (page, scenario, viewport, isReference, browserContext) =
   });
 
   await browserContext.setExtraHTTPHeaders(
-    {'MASS_BYPASS_RATE_LIMIT': process.env.MASS_BYPASS_RATE_LIMIT}
+    {'mass-bypass-rate-limit': process.env.MASS_BYPASS_RATE_LIMIT}
```
Comment (tom-fleming) [inline backstop/scripts/before.js:23]:
```suggestion
    {'Mass-Bypass-Rate-Limit': process.env.MASS_BYPASS_RATE_LIMIT}
```

File: .circleci/nightcrawler/nightcrawler.js [hunk 5]
```diff
@@ -69,6 +69,9 @@ function getAuth() {
 
 var driver = new RequestDriver({
   strictSSL: false,
+  headers: {
+    "Mass-Bypass-Rate-Limit": process.env.MASS_BYPASS_RATE_LIMIT,
```
Comment (tom-fleming) [inline .circleci/nightcrawler/nightcrawler.js]:
```suggestion
    "mass-bypass-rate-limit": process.env.MASS_BYPASS_RATE_LIMIT,
```

File: backstop/scripts/before.js [hunk 6]
```diff
@@ -20,7 +20,7 @@ module.exports = async (page, scenario, viewport, isReference, browserContext) =
   });
 
   await browserContext.setExtraHTTPHeaders(
-    {'MASS_BYPASS_RATE_LIMIT': process.env.MASS_BYPASS_RATE_LIMIT}
+    {'Mass-Bypass-Rate-Limit': process.env.MASS_BYPASS_RATE_LIMIT}
```
Comment (tom-fleming) [inline backstop/scripts/before.js]:
```suggestion
    {'mass-bypass-rate-limit': process.env.MASS_BYPASS_RATE_LIMIT}
```

File: .ddev/.env.example [hunk 7]
```diff
@@ -11,7 +11,7 @@ AKAMAI_ACCESS_TOKEN=
 AKAMAI_CLIENT_TOKEN=
 AKAMAI_CLIENT_SECRET=
 
-# Instructs Akamai disable rate limit. Used by Backstop.
+# Instructs Akamai disable rate limit. Used by Backstop and Nightcrawl.
```
Comment (tom-fleming) [inline .ddev/.env.example]:
```suggestion
# Instructs Akamai disable rate limit. Used by Backstop and Nightcrawler.
```

**Pull request merged**